fhir-model/src/main/java/org/linuxforhealth/fhir/model/string/util/StringSizeControlStrategyFactory.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.linuxforhealth.fhir.model.string.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ServiceLoader;
+
+import org.linuxforhealth.fhir.exception.FHIROperationException;
+import org.linuxforhealth.fhir.model.resource.OperationOutcome;
+import org.linuxforhealth.fhir.model.string.util.strategy.StringSizeControlStrategy;
+import org.linuxforhealth.fhir.model.type.code.IssueType;
+import org.linuxforhealth.fhir.model.util.FHIRUtil;
+
+/**
+ * Controls the creation of the StringSizeControlStrategy objects using the ServiceLoader.
+ */
+public class StringSizeControlStrategyFactory {
+    
+    private static final StringSizeControlStrategyFactory FACTORY = new StringSizeControlStrategyFactory();
```

Comment:
As discussed, service loader is a bit overkill for this. We don't need the full strategy pattern here - just an interface+class to make it easy to switch out implementations in the future should we want to.

Reply:
This has been fixed as suggested.